src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -72,15 +72,8 @@ seastar::future<> ClientRequest::start()
 	if (pg.can_discard_op(*m)) {
 	  return osd.send_incremental_map(conn.get(), m->get_map_epoch());
 	}
-	return with_blocking_future(
-	  handle.enter(pp(pg).await_map)
-	).then([this, &pg]() mutable {
-	  return with_blocking_future(
-	    pg.osdmap_gate.wait_for_map(m->get_map_epoch()));
```

Comment:
Yeah, `MOSDFastDispatchOp`  wraps `get_map_epoch()` as `get_min_epoch()` while `MOSDOp` doesn't override the fatter.

```cpp
class MOSDFastDispatchOp : public Message {
  // ...
  virtual epoch_t get_map_epoch() const = 0;
  virtual epoch_t get_min_epoch() const {
    return get_map_epoch();
  }
```